tests/FakeItEasy.Specs/EventRaisingSpecs.cs

```diff
@@ -354,7 +357,7 @@ public void FreeformReferenceTypeEventHandlerWithDerivedArguments()
 
             "When I raise the event as freeform specifying a null argument"
                 .x(() => exception = Record.Exception(() =>
-                        Fake.ValueTypeEvent += Raise.FreeForm<ValueTypeEventHandler>.With((object)null)));
+                        Fake.ValueTypeEvent += Raise.FreeForm<ValueTypeEventHandler>.With((object)null!)));

```

Comment:
but it should be cast to `object?`. Late-breaking fixup coming

Reply:
Ah, yes. If you pass null, it thinks the whole `object?[]` args is null. Waiting for your fixup, then!

Reply:
BTW, this makes me think of a possible improvement... I think we could make `With` strongly-typed, by exposing if as a property of type `TEventHandler`, instead of a method with weakly typed arguments

Reply:
Ah, no, it wouldn't work. The property would need to be of type `Func<?, ?, ..., TEventHandler>`. Pity...